app/policies/field_guide_policy.rb

```diff
@@ -6,7 +6,7 @@ def resolve(action)
     end
   end
 
-  scope :index, :show, :update, :destroy, with: Scope
+  scope :index, :show, :update, :destroy, :translate, with: Scope
```

Comment:
FieldGuides are valid targets for a Translation model to link to. This wasn't explicit anywhere in the code, and the `:translate` action wasn't listed on the call the `can_through_parent` before. This worked because ParentalControlled just passed the action along to the parent, without restricting it to the array of actions defined via `can_through_parent`. That's how this action got missed during the conversion to Pundit.

Reply:
The whitelist of valid actions could have been checked here but wasn't https://github.com/zooniverse/Panoptes/blob/b4b98f9843ec759c00f5546fd4e0be3e86b633f3/lib/role_control/parental_controlled.rb#L36